app.js

```diff
@@ -12,7 +23,58 @@ app.set('views', __dirname + '/views');
 app.use(express.static(__dirname + '/public'));
 
 // setting the spotify-api goes here:
+app.get("/", (req, res) => {
+    res.render("index")
+})
+//res.render(data.body.artists.items[0].images)
+app.get("/artist-search", (req, res) => {
+    console.log("Artist came from browser - request")
+    console.log(req.query.artist)
```

Comment:
Just for good practice every time you create a PR try to remove the console.logs that are not needed, for example you can leave the ones inside the catch